tests/test_interface.rb

```diff
@@ -464,8 +475,8 @@ def test_interface_speed_change
       interface.speed = 1000
       assert_equal('1000', interface.speed)
       interface_ethernet_default(interfaces_id[0])
-    rescue RuntimeError => e
-      assert_match(/port doesn t support this speed/, e.message)
+    rescue => e
+      speed_change_disallowed(e.message)
```

Comment:
This doesn't seem like the right approach - so now you rescue ANY exception, and if it matches the expected error message, you report a skip, and otherwise (exception does not match) you just plain pass the test?